packaging/language/composer.py

```diff
@@ -116,6 +132,8 @@ def parse_out(string):
     return re.sub("\s+", " ", string).strip()
 
 def has_changed(string):
+    if string == "":
```

Comment:
I would remove it. composer init would return changed=True which is what we want. So please remove it.